cmd/apiserver_receive_adapter/main.go

```diff
@@ -60,25 +66,58 @@ type envConfig struct {
 	Kind          StringList `required:"true"`
 	Controller    []bool     `required:"true"`
 	LabelSelector StringList `envconfig:"SELECTOR" required:"true"`
+	Name          string     `envconfig:"NAME" required:"true"`
+	// MetricsConfigBase64 is a base64 encoded json string of
+	// metrics.ExporterOptions. This is used to configure the metrics exporter
+	// options, the config is stored in a config map inside the controllers
+	// namespace and copied here.
+	MetricsConfigBase64 string `envconfig:"K_METRICS_CONFIG" required:"true"`
```

Comment:
Probably a foolish question, but why are we base64 encoding these? Flags tend to make sense, as it can be very annoying to get the escaping just right, but these are environment variables, where we don't have that problem.

Reply:
My rationale was to be consistent with knative-gcp in [here](https://github.com/google/knative-gcp/blob/master/cmd/pubsub/receive_adapter/main.go#L54). @n3wscott What's you take on this?